admin/tests/acceptance/authenticated/trainings/training_test.js

```diff
@@ -63,6 +63,25 @@ module('Acceptance | Trainings | Training', function (hooks) {
       });
     });
 
+    test('should be redirected to training detail page after training creation', async function (assert) {
+      // when
+      await visit(`/trainings/list`);
+      await clickByName('Nouveau contenu formatif');
+      await fillByLabel('Titre', 'Nouveau contenu formatif');
+      await fillByLabel('Lien', 'http://www.example.net');
+      await click(screen.getByText('Webinaire'));
+
+      await fillByLabel('Jours (JJ)', 1);
+      await fillByLabel('Heures (HH)', 0);
+      await fillByLabel('Minutes (MM)', 0);
+      await click(screen.getByText('Francophone (fr)'));
+      await fillByLabel('Nom du fichier du logo éditeur', 'Logo.svg', { exact: false });
+      await fillByLabel("Nom de l'éditeur", 'Editeur', { exact: false });
+      await clickByName('Créer le contenu formatif');
```

Comment:
Pour plus d'accessibilité, je propose : 
```suggestion
      await click(screen.getByRole('button',  { name: 'Créer le contenu formatif'}));
```